### ISSUE TITLE
Possible new link

### ISSUE THREAD

**Issue by eriktrautman** [TheOdinProject/curriculum#1431]
See email pitch below, need to see if this is actually a good link or just spam:

Dear The Odin Project.com
 
I was searching for some articles about Cloud Computing today and I came across this page:
http://www.theodinproject.com/web-development-101/the-cloud-hosting-and-software-as-a-service-saas
 
I noticed that you link to one of my favorite articles--The Beginner's Guide to the Cloud – Mashable. Linked under: “Assignment (1. Check out this explanation of cloud computing from mashable”.
 
Just wanted to give you a heads up that I created a similar piece of content. It's like The Beginner's Guide to the Cloud, but more thorough and up to date:
https://www.workbooks.com/what-is-cloud-computing
 
Might be worth a mention on your page.
 
Either way, keep up the awesome work!

**Comment by siromivel** [comment]
It looks like workbooks.com's primary objective is selling their flavor of CRM software which to me is good enough reason to not use the article. The two articles present fairly similar information, although I felt like the Mashable article is more concise and is more well written/edited. Neither present anything particularly technical, just a brief introduction to the idea of cloud computer which is really all that is necessary for that stage of the curriculum.

tl;dr: Keep the Mashable link, the new one is spammy.

**Comment by jacknguyen** [comment]
I agree the article doesn't add much depth except for the intro section. It goes briefly into the origination of cloud computing, which is interesting but the rest of the article is very B2B pitchy.

**Comment by onebree** [comment]
If you want to know cloud computing, IEEE has a free course on, I think, edX
